solutions/instances/README.md

```diff
@@ -0,0 +1,10 @@
+# IBM Observability Instance Deployable Architecture
+
+This architecture creates an observability instance on IBM Cloud®  and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An observability instance which contains Log Analysis, Cloud Monitoring and Activity Tracker instance.
```

Comment:
That wording is wrong - these are all individual instances, so just list them out individually.

Reply:
Also be careful here, this DA will not provision an Activity Tracker instance. It provisions an ATracker route to send logs to a COS target